src/actions/items.js

```diff
@@ -0,0 +1,6 @@
+export const  getPosts = ()=>{
+    return {type:'FETCH_ALL',payload:[]};
+}
+export const addPost =(data)=>{
+    return {type:'CREATE_ITEM',payload:data};
```

Comment:
Define constants in another file. (Here extract CREATE_ITEM for another file constants.js/actionTypes.js)